From f41fee8cad5bcb2538f51c38633d743978a2468a Mon Sep 17 00:00:00 2001
Message-ID: <f41fee8cad5bcb2538f51c38633d743978a2468a.1705854493.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1705854493.git.sharlatanus@gmail.com>
References: <cover.1705854493.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sat, 20 Jan 2024 20:56:13 +0000
Subject: [PATCH 19/19] gnu: libnova: Use G-Expressions.

* gnu/packages/astronomy.scm (libnova): Adjust package style by applying
G-Expressions and fix indention.
[arguments]<#:phases>: Remove trailing #f from lambda.

Change-Id: I63d239464fd3cb166449ef8904b5fd72326cb853
---
 gnu/packages/astronomy.scm | 32 ++++++++++++++++----------------
 1 file changed, 16 insertions(+), 16 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index 99b22fa39e..400ce933b3 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -3053,24 +3053,24 @@ (define-public libnova
     (name "libnova")
     (version "0.16")
     (source
-      (origin
-        (method git-fetch)
-        (uri (git-reference
-               (url "https://git.code.sf.net/p/libnova/libnova.git")
-               (commit (string-append "v" version))))
-        (file-name (git-file-name name version))
-        (sha256
-         (base32
-          "0icwylwkixihzni0kgl0j8dx3qhqvym6zv2hkw2dy6v9zvysrb1b"))))
+     (origin
+       (method git-fetch)
+       (uri (git-reference
+             (url "https://git.code.sf.net/p/libnova/libnova.git")
+             (commit (string-append "v" version))))
+       (file-name (git-file-name name version))
+       (sha256
+        (base32
+         "0icwylwkixihzni0kgl0j8dx3qhqvym6zv2hkw2dy6v9zvysrb1b"))))
     (build-system gnu-build-system)
     (arguments
-     `(#:phases
-       (modify-phases %standard-phases
-         (add-after 'unpack 'patch-git-version
-           (lambda _
-             (substitute* "./git-version-gen"
-               (("/bin/sh") (which "sh")))
-             #t)))))
+     (list
+      #:phases
+      #~(modify-phases %standard-phases
+          (add-after 'unpack 'patch-git-version
+            (lambda _
+              (substitute* "./git-version-gen"
+                (("/bin/sh") (which "sh"))))))))
     (native-inputs
      (list autoconf automake libtool))
     (synopsis "Celestial mechanics, astrometry and astrodynamics library")
-- 
2.41.0

